lib/dialects/postgres/data-types.js

```diff
@@ -163,15 +127,17 @@ module.exports = BaseTypes => {
     return 'TEXT';
   };
 
-  BaseTypes.TEXT.types.postgres = {
-    oids: [25],
-    array_oids: [1009]
-  };
+  BaseTypes.TEXT.types.postgres = ['text'];
 
-  BaseTypes.CITEXT.types.postgres = {
-    oids: [],
-    array_oids: []
-  };
+  function CITEXT() {
+    if (!(this instanceof CITEXT)) return new CITEXT();
+    BaseTypes.CITEXT.apply(this, arguments);
+  }
+  inherits(CITEXT, BaseTypes.CITEXT);
+
+  CITEXT.parse = value => value;
```

Comment:
Can be removed

Reply:
[Here](https://github.com/javiertury/sequelize/blob/b9f94a488014b4cac57fd49b2f23cf2582130626/lib/dialects/abstract/connection-manager.js#L51-L57), abstract connection manager requires a parse function to refresh a dataType. Without this CITEXT doesn't have an explicit parser, falls back to `node-pg-types` and fails the tests.